src/server/session.ts

```diff
@@ -335,7 +335,7 @@ namespace ts.server {
         private getTypeDefinition(line: number, offset: number, fileName: string): protocol.FileSpan[] {
             const file = ts.normalizePath(fileName);
             const project = this.projectService.getProjectForFile(file);
-            if (!project) {
+            if (!project || project.languageServiceDiabled) {

```

Comment:
We still need the `getProjectInfo` call to work, because the client needs to know what files are in the project and is the language service of the project disabled or not. So it's not good idea to never return the project when the language service was disabled.